README.rst

```diff
@@ -102,3 +102,34 @@ and adds a link to it in the header of any other view:
   <link rel="amphtml" href="${context/absolute_url}/@@amp">
 
 The body text is processed to remove invalid elements or to replace them by the corresponding AMP components.
+
+Tracking pixels
+^^^^^^^^^^^^^^^
+
+AMP implements tracking pixels using the ``<amp-pixel>`` tag;
+to add support for this feature you need to write an adapter that implements the ``IAMPPixelProvider`` interface like this:
+
+.. code-block:: xml
+
+    <adapter factory="my.package.adapter.AMPPixelProvider" />
+
+
+.. code-block:: python
+
+    from collective.behavior.amp.behaviors import IAMP
+    from collective.behavior.amp.interfaces import IAMPPixelProvider
+    from zope.component import adapter
+    from zope.interface import implementer
+
+    @implementer(IAMPPixelProvider)
+    @adapter(IAMP)
+    class AMPPixelProvider(object):
```

Comment:
I see an issue here. You're asking for users of this add-on to implement an adapter specifically for `IAMP`. This way it's not possible to have different tracking pixels for different content types. 

This is a documentation issue only, I believe, because the code in the `AMPView.pixel` just looks for an adapter for `self.context`. So one could write one adapter for `INewsItem` and another for `IDocument` and it would work. I haven't tested though.

Now a suggestion:

Since the adapter is for `IAMP`, which will always be implemented by a content item, and the purpose of the adapter is to spit some HTML, change it to a view. Ask add-on users to implement a view for `IAMP` or for an specific content type interface, with an specific name, like `amp-tracking-pixel`.

This approach has the following advantages:

- More newbie friendly (views are more common than adapters).
- Can implement using only ZCML + a `.pt` file.
- Allows to use the layer mechanism, for example: to enable the view only when my add-on is installed.
- Feels more natural to me. This is the main reason, the other ones are bullshit :wink: 

Please remember I'm only adding my 2 cents because you asked for feedback and this suggestion is kind of subjective. I'm not involved in the development of this add-on.